cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -365,3 +366,61 @@ def _cleanup():
         cleanup_target(provider, migrated_vm)
 
     assert src_vm_obj.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.tier(2)
+def test_osp_if_no_password_is_exposed_in_logs_during_migration(appliance, provider, request,
+                                                                mapping_data_vm_obj_mini):
+    """
+    title: OSP: Test if no password is exposed in logs during migration
+
+    Polarion:
+        assignee: sshveta
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for Vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1. Mapping created and visible in UI
+            2.
+            3. logs should not show password during migration
+    """
+    cred = []
+    cred.append(credentials["vsphere6-nested"]["password"])
+    cred.append(credentials["vmware-ssh-key"]["password"])
+    cred.append(credentials["host_default"]["password"])  # same for both RHV and OSP
+    cred.append(credentials["cloudqe-osp13-ims-rsa"]["password"])
+
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    migration_plan = migration_plan_collection.create(
+        name=fauxfactory.gen_alphanumeric(start="plan_"),
+        description=fauxfactory.gen_alphanumeric(start="plan_desc_"),
+        infra_map=mapping_data_vm_obj_mini.infra_mapping_data.get("name"),
+        vm_list=mapping_data_vm_obj_mini.vm_list,
+        target_provider=provider
+    )
+
+    assert migration_plan.wait_for_state("Started")
+    assert migration_plan.wait_for_state("In_Progress")
+    assert migration_plan.wait_for_state("Completed")
+    assert migration_plan.wait_for_state("Successful")
+
+    # Check log files for any exposed password
+    evm_log = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; grep -inr password log/evm.log").output
+    automation_log = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; grep -inr password log/automation.log").output
+
+    assert not any([True for psw in cred if psw in evm_log])
+    assert not any([True for psw in cred if psw in automation_log])
+
+    src_vm = mapping_data_vm_obj_mini.vm_list[0]
+    migrated_vm = get_migrated_vm(src_vm, provider)
+
+    @request.addfinalizer
+    def _cleanup():
+        cleanup_target(provider, migrated_vm)
```

Comment:
Clean up infra mapping and migration plan too

Reply:
Mapping should gets deleted by fixture mapping_data_vm_obj_mini itself, So adding cleanup for migration plan only